keepassxc-browser/content/totp-field.js

```diff
@@ -25,7 +27,7 @@ class TOTPFieldIcon extends Icon {
 };
 
 TOTPFieldIcon.prototype.initField = function(field) {
-    if (!field || field.getAttribute('kpxc-totp-field') === 'true') {
+    if (!field || field.getAttribute('kpxc-totp-field') === 'true' || field.offsetWidth < MINIMUM_SIZE) {
```

Comment:
We should also check to maximum input length of the field, if it is less than 4 ignore it.

Reply:
Also I am personally not satisfied with absolute number checks like this because you could (in theory) have a small field size with a small font.

Reply:
Me neither. But then again, the GOG site here doesn't use the input length at all, so that must be also checked just in case, even if it's not 100% reliable.